presqt/api_v1/tests/test_init.py

```diff
@@ -0,0 +1,15 @@
+from django.test import TestCase, Client
```

Comment:
There is a rest_framework specific test client.
I think that it has some built-in advantages when testing API endpoints.

It looks like you used this `test_target` already. :)